contracts/HolographOperator.sol

```diff
@@ -1062,6 +1075,26 @@ contract HolographOperator is Admin, Initializable, HolographOperatorInterface {
     }
   }
 
+  /**
+   * @notice Get the Minimum Gas Price
+   * @dev This amount is used as the value that will define a job as underpriced is lower than
```

Comment:
The minimum value required to execute a job without being under priced

Reply:
Is this a suggested alternative @dev text?

Reply:
yeah i guess so. just found the original one a little unclear

Reply:
Ok perfect, will update it to this text then.